build-system/tasks/release/index.js

```diff
@@ -121,9 +121,6 @@ async function prepareEnvironment_(outputDir, tempDir) {
   await fs.emptyDir(outputDir);
   await fs.emptyDir(tempDir);
   logSeparator_();
-
```

Comment:
Remove "ensures dependencies are installed" from the jsdoc

Reply:
Good catch. Done.